packages/website/src/pages/related-tools.tsx

```diff
@@ -2,96 +2,183 @@ import CarbonAd from 'src/components/CarbonAd'
-    link: 'https://github.com/CoenWarmer/traymoji',
-  },
-  {
-    name: 'alfred-gitmoji',
-    description: 'Gitmoji Workflow for Alfred',
-    link: 'https://github.com/techouse/alfred-gitmoji',
-  },
-  {
-    name: 'gitmojiapp',
-    description: 'A Flutter Gitmoji App for macOS, Linux, and Windows',
-    link: 'https://github.com/patrick-fu/GitmojiApp',
-  },
-  {
-    name: 'githubmoji',
-    description:
-      'A Firefox addon that adds a predictive gitmoji picker to GitHub commit message inputs.',
-    link: 'https://github.com/ma1ted/githubmoji',
-  },
-  {
-    name: 'gitmoji-changelog-action',
-    description: 'GitHub Action for gitmoji-changelog',
-    link: 'https://github.com/sercanuste/gitmoji-changelog-action',
-  },
-  {
-    name: 'raycast-gitmoji-search',
-    description: 'Gitmoji extension for Raycast',
-    link: 'https://www.raycast.com/ricoberger/gitmoji',
-  },
-  {
-    name: 'gitmoji-fuzzy-hook',
-    description:
-      'Fuzzy finder git-commit-hook for prepending a gitmoji (cmd & GUI).',
-    link: 'https://gitlab.com/raabf/gitmoji-fuzzy-hook',
-  },
-  {
-    name: 'genmoji',
-    description: 'Gitmoji commit message generation using AI',
-    link: 'https://github.com/segersniels/genmoji',
-  },
+	{
+		emoji: '📦',
+		title: 'Apps',
+		items: [
+			{
+				name: 'gitimoji',
+				description: 'A Gitmoji App for macOS',
+				link: 'https://github.com/TimoZacherl/gitimoji',
+			},
+			{
+				name: 'gitmojiapp',
+				description: 'A Flutter Gitmoji App for macOS, Linux, and Windows',
+				link: 'https://github.com/patrick-fu/GitmojiApp',
+			},
+			{
+				name: 'gitmojiBar',
+				description: 'MacOS menubar for easy copy gitmoji',
+				link: 'https://github.com/techinpark/gitmojiBar',
+			},
+			{
+				name: 'traymoji',
+				description: 'A Electron Tray App for Gitmojis',
+				link: 'https://github.com/CoenWarmer/traymoji',
+			},
+		],
+	},
+	{
+		emoji: '🏭',
+		title: 'Automatic generators',
+		items: [
+			{
+				name: 'genmoji',
+				description: 'Gitmoji commit message generation using AI',
+				link: 'https://github.com/segersniels/genmoji',
+			},
+			{
+				name: 'gitmoji-changelog',
+				description: 'A changelog generator for gitmoji',
+				link: 'https://github.com/frinyvonnick/gitmoji-changelog/',
+			},
+			{
+				name: 'gitmoji-changelog-action',
+				description: 'GitHub Action for gitmoji-changelog',
+				link: 'https://github.com/sercanuste/gitmoji-changelog-action',
+			},
+			{
+				name: 'gitmoji-semver',
+				description: 'Just commit with gitmoji, and you’ll get auto versioning by semantic versioning and auto release to github',
+				link: 'https://github.com/nkmr-jp/gitmoji-semver',
+			},
+		],
+	},
+	{
+		emoji: '🌐',
+		title: 'Browser extensions',
+		items: [
+			{
+				name: 'githubmoji',
+				description: 'A Firefox addon that adds a predictive gitmoji picker to GitHub commit message inputs',
+				link: 'https://addons.mozilla.org/firefox/addon/githubmoji',
```

Comment:
We do link to github repos always so let's keep the original content the author added:

```suggestion
				link: 'https://github.com/ma1ted/githubmoji',
```